app/utils/CommonHelper.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018 The Delphi Team.
+// See the LICENCE file distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package utils
+
+trait CommonHelper {
+    def addHttpProtocolIfNotExist(url: String): String ={
+      var tempUrl:String = url
+      val exist = url.contains("http://")
```

Comment:
This will not match https:// based URLs. Please adapt.

Reply:
This has been fixed, the check now checks for either "https://" or "http://".

Reply:
Ah sorry to reiterate here... but why is this using `contains` and not `startsWith` ?

Reply:
Good point, i found no good reason to prefer ```contains``` over ```startsWith``` here, so i replaced it in the latests commit.